modules/claims_api/app/workers/claims_api/claim_auditor.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'sentry_logging'
+
+module ClaimsApi
+  class ClaimAuditor
+    include Sidekiq::Worker
+    include SentryLogging
+
+    def perform
+      return unless Settings.claims_api.audit_enabled
+
+      report_threshold = Settings.claims_api.claims_pending_reporting.threshold
+      claims = ClaimsApi::AutoEstablishedClaim.where(status: ClaimsApi::AutoEstablishedClaim::PENDING)
+                                              .where('created_at < ?', (report_threshold / 1000).seconds.ago)
+      return unless claims.any?
+
+      client = SlackNotify::Client.new(webhook_url: Settings.claims_api.slack.webhook_url,
+                                       channel: '#api-benefits-claims',
+                                       username: 'ClaimAuditor')
```

Comment:
Any value in moving these 2 values into the `settings.yml` file?

Reply:
eh, maybe the channel but i went this route since i was already adding a ton to the settings file and these should always be the same for this job.

i could see moving the channel out to settings in the future, but i think this is good enough for now